src/backend/marsha/core/lti.py

```diff
@@ -87,7 +88,18 @@ def verify(self):
         ):
             raise LTIException()
 
-        return passport.consumer_site or passport.playlist.consumer_site
+        consumer_site = passport.consumer_site or passport.playlist.consumer_site
+
+        # Make sure we only accept requests from domains in which the "top parts" match
+        # the URL for the consumer_site associated with the passport.
+        # eg. sub.example.com & example.com for an example.com consumer site.
+        domain_check = urlparse(self.request.META.get("HTTP_REFERER")).hostname
+        if domain_check != consumer_site.domain and not domain_check.endswith(
```

Comment:
Checking that `domain_check != consumer_site.domain` seems useless since:
```shell
>>> "example.com".endswith("example.com")
True
```

Reply:
This is not what my check looks like. It's `.endswith(".{:s}".format(consumer_site.domain))`

Reply:
ok sorry I didn't notice the "." added !